network-api/networkapi/wagtailpages/pagemodels/buyersguide/taxonomies.py

```diff
@@ -9,14 +11,42 @@ class BuyersGuideContentCategory(wagtail_models.TranslatableMixin, models.Model)
     title = models.CharField(max_length=100, null=False, blank=False)
     slug = models.SlugField(max_length=100, null=False, blank=True, unique=True)
 
+    panels = [admin_panels.FieldPanel("title")]
+
     class Meta(wagtail_models.TranslatableMixin.Meta):
         verbose_name = "Buyers Guide Content Category"
         verbose_name_plural = "Buyers Guide Content Categories"
 
     def __str__(self) -> str:
         return self.title
 
-    def save(self, *args, **kwargs) -> None:
-        if not self.slug:
-            self.slug = text_utils.slugify(self.title)
-        super().save(*args, **kwargs)
+    def clean_slug(self):
```

Comment:
**Kudos** Just wanted to leave kudos for a really well documented function 👍 I also appreciate that this is being done it its own `clean_slug` function instead of the `clean` method itself! 


**Question**

While testing this method out in the CMS, I can confirm that translation seems to work. However, I do notice that I get the following warning after publishing. Was wondering if this is to be expected?

![image](https://user-images.githubusercontent.com/18314510/208560525-40023e5a-a933-47a9-a808-02bc5f6c3795.png)

Reply:
Ah good catch! Thanks @danielfmiranda I had missed this. I believe this is because the translatable fields are not specified explicitly. I will add that.